plugins/samplesource/CMakeLists.txt

```diff
@@ -78,7 +78,7 @@ else(LIBUSB_FOUND AND LIBMIRISDR_FOUND)
 endif(LIBUSB_FOUND AND LIBMIRISDR_FOUND)
 
 find_package(SoapySDR)
-if(LIBUSB_FOUND AND SOAPYSDR_FOUND)
+if(SOAPYSDR_FOUND)
```

Comment:
Fine but if this is the only change I'd rather patch this file than do a merge

Reply:
Many devices when using Soapysdr do not require libusb.
Libusb preinstalled in Soapyusdr.

Reply:
The same is required for  plugins\samplesink\CMakeLists.txt
line 32   if(LIBUSB_FOUND AND SOAPYSDR_FOUND)